fw/vhost.c

```diff
@@ -2154,8 +2163,12 @@ tfw_cfgop_frang_body_len(TfwCfgSpec *cs, TfwCfgEntry *ce)
 {
 	int r;
 	FrangVhostCfg *cfg = tfw_cfgop_frang_get_cfg();
+	long deflt;
 
-	if (ce->dflt_value && cfg->http_body_len)
+	if ((r = tfw_cfg_parse_long(cs->deflt, &deflt)))
+		return r;
+
+	if (ce->dflt_value && cfg->http_body_len != deflt)
```

Comment:
Here we trying to not override values that not specified in new `frang_limits` by default values. Please describe it in commit message. And also, looks like if `ce->dflt_value` is false we can skip default value parsing. 

@krizhanovsky FYI it looks controversial to have two different `deflt` values for single directive in different scopes.

Reply:
Here he try to not override values that not specified in new frang_limits (default values) by values specified in top level frang config